Makefile

```diff
@@ -118,5 +118,8 @@ env-up:                         ## Start development environment.
 env-down:                       ## Stop development environment.
 	docker-compose down --volumes --remove-orphans
 
+setup-dev:
```

Comment:
Here I think we should depend on `install` and use `pmm-agent`. Please add a doc comment – it is visible on `make` without arguments.

Reply:
ok

Reply:
why do we should depend on install?

Reply:
> and use pmm-agent

To run the latest installed pmm-agent – to avoid ldflags copy&pasting.

Reply:
but I run it by `go run main.go` here.

Reply:
> to avoid ldflags copy&pasting.

Reply:
Look, I don't mind much. I just don't see a competing reason to make it different from `run` / `run-race`.

Reply:
ah, ok, got you